### ISSUE TITLE
A11y_AzureDataStudio_JuypyterBook_ScreenReader:  NVDA doesn’t announce selected state after selecting the tree item such as “TSG021 - Get Kubernetes cluster information” .

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7796]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#MAS1.3.1;MAS4.1.2;#DesktopApp;#JAWS;#NVDA;#SQL AzureDataStudio;#Win32

**Environment Details:	**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA,JAWS

**Steps to Reproduce:**
1.Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide" 
5.Navigate to the left section of jupyter books under the untitled books and click on "save" icon .
6.Verify that all the elements of "Save juypyter book" are accessible and MAS Compliant.
7.Start NVDA.	
8.Navigate till  tree item using arrows and select that item.
9.Observe the announcement by screen reader. 
 
**Actual:**
NVDA doesn’t announce selected state after selecting the tree item such as “TSG021 - Get Kubernetes cluster information” .	
	
**Expected:** 
NVDA should announce selected state after selecting the tree item for example  “TSG021 - Get Kubernetes cluster information selected” .

**User Impact:**
Screen reader user will it difficult to navigate the screens as they won’t understand that particular tree item is selected or not.

**Attachment for Reference:**
[7796_A11y_AzureDataStudio_JuypyterBook_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742621/7796_A11y_AzureDataStudio_JuypyterBook_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by MaddyDev** [comment]
Created https://github.com/microsoft/vscode/issues/88800 to track it.

**Comment by hannahqin** [comment]
Closing as a duplicate of #7795

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
